dashboard/package.json

```diff
@@ -64,7 +63,6 @@
     "@playwright/test": "^1.24.2",
     "@trivago/prettier-plugin-sort-imports": "^3.3.0",
     "@types/amplitude-js": "^8.16.2",
-    "@types/flux": "^3.1.11",
```

Comment:
`universal-cookie` is also not used

Reply:
kinda, it is there and in the code, but we should remove it... we can create a task and do it as a follow-up. The good thing is that we don't need the node_modules from the dashboard build so that's not sizing up the final image.